TestProjects/UniversalGraphicsTest_Foundation/Assets/Test/TestFilters/TestCaseFilters.asset

```diff
@@ -154,3 +154,13 @@ MonoBehaviour:
     XrSdk: 
     StereoModes: 0
     Reason: Decals not supported
+  - FilteredScene: {fileID: 0}
+    FilteredScenes:
+    - {fileID: 102900000, guid: 6e7a83866a1e446b3becf7972371c5d0, type: 3}
+    - {fileID: 102900000, guid: 773767ac5b09198408c0c084dc243abc, type: 3}
+    ColorSpace: -1
+    BuildPlatform: 13
+    GraphicsDevice: 11
+    XrSdk: 
+    StereoModes: 7
+    Reason: 'XR : shader error during build'
```

Comment:
I think Dennis got a bug in on this, can we refence that # here?

Reply:
I've reverted this change because it still fails the build. We'll follow up on the bug.